hi-ml-histopathology/src/histopathology/configs/classification/DeepSMILEPanda.py

```diff
@@ -94,7 +116,171 @@ def get_data_module(self) -> PandaTilesDataModule:
         )
 
     def get_slides_dataset(self) -> Optional[PandaDataset]:
-        return PandaDataset(root=self.local_datasets[1])                             # type: ignore
+        return PandaDataset(root=self.local_datasets[1])  # type: ignore
+
+
+class DeepSMILETilesPanda(DeepSMILETilesPandaBasic):
+    """
+    Configuration for PANDA experiments from Myronenko et al. 2021:
+    (https://link.springer.com/chapter/10.1007/978-3-030-87237-3_32)
+
+    `is_finetune` sets the fine-tuning mode. For fine-tuning,
+    batch_size = 2 runs on 8 GPUs with
+    ~ 6:24 min/epoch (train) and ~ 00:50 min/epoch (validation).
+    """
+    def __init__(self, **kwargs: Any) -> None:
+        default_kwargs = dict(
+            pool_type=AttentionLayer.__name__,
+            num_transformer_pool_layers=4,
+            num_transformer_pool_heads=8,
+            pool_hidden_dim=2048,
+            encoding_chunk_size=60,
+            max_bag_size=56,
+            batch_size=8,  # effective batch size = batch_size * num_GPUs
+            max_epochs=50,
+            l_rate=3e-4,
+            weight_decay=0,
+            primary_val_metric=MetricsKey.ACC)
+        default_kwargs.update(kwargs)
+        super().__init__(**default_kwargs)
+
+    def setup(self) -> None:
+        # Params specific to transformer pooling
+        if self.pool_type in [TransformerPooling.__name__, TransformerPooling.__name__]:
+            self.l_rate = 3e-5
+            self.weight_decay = 0.1
+        # Params specific to fine-tuning
+        if self.is_finetune:
+            self.batch_size = 2
+        super().setup()
+
+    def get_transforms_dict(self, image_key: str) -> Dict[ModelKey, Union[Callable, None]]:
+        if self.is_caching:
+            transform_train = Compose([
+                                      LoadTilesBatchd(image_key, progress=False),
+                                      ScaleIntensityRanged(keys=image_key, a_min=0.0, a_max=255.0),
+                                      EncodeTilesBatchd(image_key, self.encoder, chunk_size=self.encoding_chunk_size)
+                                      ])
+        else:
+            # Use same transforms as demonstrated in
+            # https://github.com/Project-MONAI/tutorials/blob/master/pathology/multiple_instance_learning/panda_mil_train_evaluate_pytorch_gpu.py
+            transform_train = Compose([
+                                      LoadTilesBatchd(image_key, progress=False),
+                                      transform_dict_adaptor(RandomHorizontalFlip(p=0.5), image_key, image_key),
+                                      transform_dict_adaptor(RandomVerticalFlip(p=0.5), image_key, image_key),
+                                      transform_dict_adaptor(RandomRotationByMultiplesOf90(), image_key, image_key),
```

Comment:
Has this config been tested? I believe ` transform_dict_adaptor(RandomRotationByMultiplesOf90(), image_key, image_key)` is throwing a pickle error. We discussed to try out the MONAI or torch transform implementation to avoid it.

Reply:
@vale-salvatelli  this is the TilesDeepSmilePanda config where it works. IT was throwing an error on the Slides config, which is now changed (under review in cpath)